src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const calculator = {
+    operate(callback, value) {
+      callback(value);
+
+      return calculator;
+    },
+    add(value) {
+      result += value;
+
+      return calculator;
+    },
+    reset() {
+      result = 0;
+
+      return calculator;
+    },
+    subtract(value) {
+      result -= value;
+
+      return calculator;
+    },
+    divide(value) {
```

Comment:
You are not handling division by zero. This will lead to Infinity, which might not be the expected behavior. You should add a check to ensure the value is not zero before performing the division.